### PR TITLE
env.sh -> env.sh.sample;  Refactor of calendar script; Move e-Paper lib dependency to git submodule

### PR DESCRIPTION
**Pull request by lifeofbrian**
Setting up the default sample env.sh as a separate file, more productive in managing changes

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,5 +7,4 @@ env.sh
 .vscode
 *.pickle
 credentials*
-env.sh
```
Comment (mendhak) [inline .gitignore:10]: env.sh should stay in .gitignore, so that any changes made by users don't get committed, it just stays untracked.

Comment (lifeofbrian) [reply .gitignore:10]: I removed this because there was two entries for env.sh in the .gitignore.

File: run.sh [hunk 2]
```diff
@@ -28,12 +25,5 @@ fi
 
 inkscape screen-output-weather.svg --without-gui -e screen-output.png -w$WAVESHARE_WIDTH -h$WAVESHARE_HEIGHT --export-dpi=300
 
-# Convert to a black and white, 1 bit bitmap
-#convert -colors 2 +dither -type Bilevel -monochrome screen-output.png screen-output.bmp
-
-
-SHOULD_REFRESH=0
-current_minute=`date +"%M"`
```
Comment (mendhak) [inline run.sh:36]: Oh yeah, the old stuff.

**Review by mendhak — APPROVED** [review]
Ah Github has a review feature.  Just trying this out.

**Pull request merged**